packages/fx-core/resource/strings.json

```diff
@@ -19,13 +19,31 @@
     "CancelProvision": "CancelProvision",
     "AlreadyProvisionNotice": "'%s' is already provisioned in the cloud, will create the teams app again if it has been removed.",
     "NotProvisionedNotice": "'%s' is not provisioned in the cloud yet. Please provision first.",
+
     "GenerateArmTemplateFailNotice": "'%s' Failed to create ARM template for required Azure services.",
-    "StartDeployArmTemplateNotice": "[%s] Start Deploying Arm templates.",
-    "SucessfullyCompileBicepNotice": "[%s] Successfully compile bicep files to JSON.",
-    "SucessfullyDeployArmTemplateNotice": "[%s] Successfully deploy Arm templates. Resource group name: %s. Deployment name: %s",
-    "FailedToDeployArmTemplateNotice": "[%s] Failed to deploy Arm templates. Resource group name: %s. Deployment name: %s",
-    "PollDeploymentStatusNotice": "[%s] Polling deployment status...",
-    "EndDeployArmTemplateNotice": "[%s] Successfully deployed Arm templates."
+
```

Comment:
cc @MuyangAmigo to help review the wording here. @MuyangAmigo you may focus on this file only.